core/controllers/domain_objects_validator.py

```diff
@@ -243,3 +243,49 @@ def validate_suggestion_images(files):
     # The files argument do not represent any domain class, hence dict form
     # of the data is returned from here.
     return files
+
+
+def validate_exploration_change(exp_change_dict):
+    """Validate the exploration change list
```

Comment:
Every docstring should end with a period.

Reply:
Done, sorry about that.
Thanks